core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCommits.java

```diff
@@ -206,6 +191,89 @@ public void testFailedCommit() throws Exception {
     Assertions.assertThat(manifests).as("Should contain 0 Avro manifest files").isEmpty();
   }
 
+  @Test
+  public void testCommitFailedBeforeChangeVersionHint() throws Exception {
```

Comment:
Spacing in this table is very difficult to read

Reply:
Sorry, what's mean?
Do you mean that the method's naming is too long for easy reading?